src/okta/components/callback.component.ts

```diff
@@ -10,28 +10,33 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { Component, OnInit, Optional, Injector } from '@angular/core';
-import { Router } from '@angular/router';
-import { OktaAuthService } from '../services/okta.service';
+import { Component, OnInit, Optional, Injector, Inject } from '@angular/core';
+import { OktaAuth } from '@okta/okta-auth-js';
```

Comment:
This means we have a real dependency on `@okta/okta-auth-js`, it should not be removed from dependencies in package.json

Reply:
Similar decision has been made on `okta-react`, and we eventually keeps `okta-auth-js` as a peerDeps in okta-react.

We probably can think this way, `react` or `angular` is used in the SDKs, but they have been listed in the peerDeps list instead of `dependencies`. In this way, devs will have the flexibility to install the okta-auth-js version that the SDK supports.

Reply:
It is a real dependency though. Elsewhere, the method `toRelativeUrl` is being called from an import. This is not from the instance passed in by dev.